test/apiv2/20-containers.at

```diff
@@ -174,6 +174,9 @@ if root; then
     podman rm -f $CTRNAME
 fi
 
+# Container create without existing image should return 404
+t POST libpod/containers/create Image="foo" 404
```

Comment:
LGTM. Minor suggestion, for readability and test strictness, but only should you need to repush:
```diff

 # Container create without existing image should return 404
-t POST libpod/containers/create Image="foo" 404
+t POST libpod/containers/create Image="nosuchimage" 404 \
+  .cause="image not known"
 
```